app/frontend/src/components/designer/FormViewer.vue

```diff
@@ -62,40 +67,47 @@
           "
           transition="scale-transition"
         >
-          <div v-if="saving">
+          <div v-if="saving" :class="[isRTL ? 'mr-1' : '']">
```

Comment:
For consistency do we also want to do these as `:class="{ 'mr-1': isRTL }"`? I ask because when I see this I wonder if `:class` treats the empty string as no class to add, or what it does...